src/status_im/contexts/profile/push_notifications/effects.cljs

```diff
@@ -1,60 +1,130 @@
 (ns status-im.contexts.profile.push-notifications.effects
   (:require
     [native-module.push-notifications :as native-module.pn]
+    [promesa.core :as promesa]
     [react-native.platform :as platform]
-    [react-native.push-notification-ios :as pn-ios]
+    [status-im.common.log :as logger]
+    [status-im.config :as config]
+    [status-im.contexts.push-notifications.notifications :as pn-notifications]
+    [status-im.contexts.push-notifications.permissions :as pn-permissions]
+    [status-im.contexts.push-notifications.remote-token :as pn-remote-token]
     [utils.re-frame :as rf]))
 
-(def ios-listeners-added? (atom nil))
+(defn attempt-enable-notifications
+  []
+  (-> (pn-permissions/request-notification-permissions)
+      (promesa/then
+       (fn [permissions]
+         (if (:authorized? permissions)
+           permissions
+           (promesa/rejected
+            (ex-info "Failed to authorize push notifications"
+                     {:code        nil
+                      :description "Requested and failed to authorize permissions for push notifications"
+                      :details     {}})))))))
```

Comment:
can be removed i guess

Reply:
could you clarify a bit more, why can this be remove?

Reply:
i mean the details field with the empty map